docs/sources/configure-client/grafana-agent/ebpf/configuration.md

```diff
@@ -0,0 +1,134 @@
+---
+title: "Configuration reference"
+menuTitle: "Configuration Reference"
+description: "Grafana Agent eBPF Profiling configuration reference"
+weight: 30
+---
+
+
+## Configuration
+
+The component configures and starts a new ebpf profiling job to collect performance profiles from the current host.
+
```

Comment:
I think this would be a great place to have a diagram that shows how this works. Maybe for a second iteration but something showing the various elemeents:

- grafana agent
- eBPF program
- targets 
- forward_to
- pyroscope.write

and how these all play together would be really helpful in understanding how this works

Reply:
Hopefully that helps 
<img width="2388" alt="image" src="https://github.com/grafana/pyroscope/assets/1053421/7dbb75f0-dc53-46f7-a852-a3d051356452">